src/device-registry/controllers/event.controller.js

```
@@ -3142,6 +3142,29 @@
+      const result = await EventModel(tenant).fetch(filter, req.cohortAccess);
+
+      // Check if any private cohorts were accessed for logging/monitoring
+      if (req.cohortAccess && req.cohortAccess.canAccessPrivateCohorts) {
+        logger.info(
+          `User ${req.userId} accessed events with private cohort access`
+        );
+      }
+
+      return res.status(200).json(result);
+    } catch (error) {
+      logger.error(`Error getting events: ${error.message}`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)